modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,451 @@
+        to:
+            kind: DockerImage
+            name: {{.Values.registry}}/{{.Values.specialResourceModule.metadata.name}}-{{.Values.groupName.driverContainer}}:{{.Values.kernelFullVersion}}
+----
+.. Save this YAML template for the ACM policy in the `templates` directory as `0002-policy.yaml`.
+... For a connected environment:
++
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    name: policy-{{.Values.specialResourceModule.metadata.name}}-ds
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST-CSF
+spec:
+    remediationAction: enforce
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: config-{{.Values.specialResourceModule.metadata.name}}-ds
+            spec:
+                remediationAction: enforce
+                severity: low
+                namespaceselector:
+                    exclude:
+                        - kube-*
+                    include:
+                        - '*'
+                object-templates:
+                    - complianceType: musthave
+                        objectDefinition:
+                        apiVersion: v1
+                        kind: Namespace
+                        metadata:
+                            name: {{.Values.specialResourceModule.spec.namespace}}
+                    - complianceType: mustonlyhave
+                        objectDefinition:
+                        apiVersion: v1
+                        kind: ServiceAccount
+                        metadata:
+                            name: {{.Values.specialResourceModule.metadata.name}}
+                            namespace: {{.Values.specialResourceModule.spec.namespace}}
+                    - complianceType: mustonlyhave
+                        objectDefinition:
+                            apiVersion: rbac.authorization.k8s.io/v1
+                            kind: Role
+                            metadata:
+                                name: {{.Values.specialResourceModule.metadata.name}}
+                                namespace: {{.Values.specialResourceModule.spec.namespace}}
+                            rules:
+                                - apiGroups:
+                                    - security.openshift.io
+                                resources:
+                                    - securitycontextconstraints
+                                verbs:
+                                    - use
+                                resourceNames:
+                                    - privileged
+                    - complianceType: mustonlyhave
+                        objectDefinition:
```

Comment:
`objectDefinition` should be at the same level as `complianceType`

Reply:
corrected